readme.md

```diff
@@ -1,404 +1,61 @@
 
-### Tetris
+#### Why Use Firebase?
 
-Before we start working with Firebase and AngularFire, let's see it in action.  Visit the page for the [Tetris Example](https://www.firebase.com/tutorial/#session/gf3bu09wvlf).
+It's fast.
+* Firebase uses Websockets to maintain a constant, open connection between the client and the database. Rather than use a traditional call-and-return system of requests and responses like HTTP, the browser and server are constantly in communication with each other and are immediately aware of changes on either end. Using Firebase, we can establish three-way data binding in an Angular application between the controller, the browser and the database.
 
-Open another browser window to view the database changes, in real-time: https://gf3bu09wvlf.firebaseio-demo.com/.  Then press play.
+> If you're interested in learning more about Websockets, [start here](http://www.html5rocks.com/en/tutorials/websockets/basics/).
 
-See how the database is updated as each block lands on the board?  This database is updated in real-time to store game data.
+It's user friendly.
+* Firebase's graphical interface means we only need to load up a browser in order to explore what we have in our database.
 
-### Quickstart
+Perhaps the biggest selling point: multiple clients can easily use the same database/API.
+* It's just a matter of including the proper credentials in your application.
 
-When you have some time, I recommend you follow the [AngularFire Quickstart](https://www.firebase.com/docs/web/libraries/angular/quickstart.html).  Through that, you will create a simple html page that will let you play with AngularFire.  Right now, I'll show you what real-time, three-way binding looks like.
+There are lots of notable apps that use Firebase, including Shazam and NPR One. [There are plenty more](https://firebase.google.com/customers/).
 
-All that is possible with just the few lines of code.
+## AngularFire
 
-## Websockets
+How can we connect our Angular application to Firebase? By using **AngularFire**.
 
-To achieve this kind of performance, we are using a different connection mechanism: Websockets.  Up until this point, we have used HTTP (HypeText Transfer Protocol) which supports the familiar request/response cycle.  Using a Websocket maintains a connection between your browser and the server, allowing data to be passed back and forth.  These connections are persistent (always on), full duplex (simultaneously bi-directional) and blazingly fast.
+AngularFire is a Javascript library created by Firebase. It provides us with an intuitive means of interacting with a database not unlike ActiveRecord or ngResource. We will be using the AngularFire library to access Firebase from Angular. We'll explore how to use it by building a todo app later on in this lesson but if you'd like to learn more, [the official documentation is a good place to start](https://github.com/firebase/angularfire).
```

Comment:
nice job not using the word interface! Thats what I want to describe it as in one word